stock_prices/stock_prices.py

```diff
@@ -5,11 +5,26 @@
 def find_max_profit(prices):
   pass
 
+def find_max_profit(prices):
+    buy = prices[0]
+    sell = 0
+    profit = -1
+    for n in range(0,len(prices)-1):
+        x = prices[n]
+        if x < buy:
+            buy = x
+            sell = 0
+        elif x > sell:
+            sell = x
+            profit = sell - buy
+    return profit
+
+print (find_max_profit(([1050, 270, 1540, 3800, 2]))
```

Comment:
I think this would work if you took out line 22. Having a print statement before the argument parser causes syntax errors.